app/controllers/companies_controller.rb

```diff
@@ -3,10 +3,10 @@
 require 'gpw/client'
 
 class CompaniesController < ApplicationController
+  before_action :validate_params
```

Comment:
do you need it before all actions in this controller? To me it looks that you need it only for `index` action :thinking:

Reply:
Right, this validation I need only for index, updated